docs/about.rst

```diff
@@ -0,0 +1,31 @@
+About
+=====
+
+Disclaimer
+----------
+
+Naggum don't aim to be yet another Common Lisp or Scheme or whatever
+implementation. Instead, we are trying to deliver a modern Lisp dialect that
+makes use of most of CLI benefits.
+
+Features
+--------
+
+- low level and high level features coupled together
```

Comment:
Then I'd write that "Naggum provides both direct access to low-level features of CLI (i.e., CIL) and useful high-level primitives like functions, namespaces (do you? Will you?), flow control expressions and so forth".